frontend/Dockerfile

```diff
@@ -1,15 +1,23 @@
-FROM jfloff/alpine-python:3.6
-
+FROM femtopixel/google-closure-compiler:0.1.5 as builder
```

Comment:
Does this include also the Python functionality we need?

Reply:
we use two images if you see. this one is only for minifying the JS

Reply:
Where is the other image jfloff/alpine-python:3.6? I think I am missing what will happen with merge. Will we have two frontend images, and 2 Dockerfiles?